docs/reference/release-notes/highlights.asciidoc

```diff
@@ -13,11 +13,30 @@ For detailed information about this release, see the <<es-release-notes>> and
 
 // Use the notable-highlights tag to mark entries that 
 // should be featured in the Stack Installation and Upgrade Guide:
+
 // tag::notable-highlights[] 
-// [discrete]
-// === Heading
-//
-// Description. 
+[discrete]
+=== Better space efficiency for `keyword`, `match_only_text`, and `text` fields
+
+The inverted index switched to a new encoding that is more space-efficient.
+This change will benefit `keyword` fields, `match_only_text` fields and, to a
+lesser extent, `text` fields. On the dataset of application logs that the {es}
+team uses for nightly benchmarks, this translated into a 14.4% reduction of the
+size of the index of the `message` field (mapped as `match_only_text`) and an
+overall 3.5% reduction of the on-disk footprint.
+
+This change will be picked up automatically by both new indices, and existing
+indices for every new segment.
+
+[discrete]
+=== Faster indexing of `geo_point`, `geo_shape` and range fields
+
+Some optimizations have been made to indexing of multi-dimensional points,
+which are used by `geo_point` fields, `geo_shape` fields and range fields.
```

Comment:
Rewords this to avoid passive voice. I also note that multi-dim points are an internal data structure.
```suggestion
We've optimized indexing speeds for multi-dimensional points, an
internal data structure used for `geo_point`, `geo_shape`, and range fields.
```